src/NodeAFK.js

```diff
@@ -0,0 +1,55 @@
+const Listener = require('./Listener');
+
+class NodeAFK {
+  constructor() {
+    this.listeners = {};
+    this.nextListenerId = 1;
+  }
+
+  getAllListeners() {
```

Comment:
With `this.listeners` being a publicly accessible  property on the class, is there any need for this getter?